systems_cross_building/cross_building.rst

```diff
@@ -701,8 +701,8 @@ You can use a profile like the following to cross-build your packages for ``iOS`
 
     **ARM**
 
-    - https://developer.arm.com/docs/dui0773/latest/compiling-c-and-c-code/specifying-a-target-architecture-processor-and-instruction-set
-    - https://developer.arm.com/docs/dui0472/latest/compiler-command-line-options
+    - https://developer.arm.com/documentation/dui0773/latest/compiling-c-and-c-code/specifying-a-target-architecture-processor-and-instruction-set
+    - https://developer.arm.com/documentation/dui0472/latest/compiler-command-line-options
```

Comment:
🤔  This link fails in my browser, the one that works is https://developer.arm.com/documentation/dui0472/m/compiler-command-line-options